python_ta/typecheck/base.py

```diff
@@ -240,5 +240,49 @@ def __init__(self,
         self.nonlocals = nonlocals_ or {}
         self.globals = globals_ or {}
 
+
+    def get_locals(self):
```

Comment:
These methods are not necessary and can be removed. If absolutely necessary, external code can access the attributes directly, but you should attempt to minimize this as much as possible.